### ISSUE TITLE
Build error following gradle upgrade

### ISSUE THREAD

**Issue by msdsoftware** [Unidata/thredds#641]
I have just upgraded gradle to 3.1 on two different macOS machines, and on both got the same error when trying to do a `gradle clean build` on the thredds-master package.

```
* Where:
Script '/Users/rs/Devo/java/thredds/thredds-master/gradle/testing.gradle' line: 182

* What went wrong:
Failed to notify build listener.
> Can't find :buildSrc test result dir.. Expression: this.file(buildSrc/build/test-results/binary/test).exists()

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED
```

In both cases I was able to get the build to work by manually creating the `buildSrc/build/test-results/binary/test` directory.

This problem also occurred a month ago when I upgraded one of the machines to gradle 3.0. The other machine had been running gradle 2.14 until today.

**Comment by cwardgar** [comment]
That's due to [this change](https://docs.gradle.org/3.0/release-notes.html#unique-default-test-result-and-report-directories-for-test-tasks) in Gradle 3.0.

I'm curious though, why aren't you building thredds using the wrapper? The wrapper is intended to avoid precisely these sorts of problems.

**Comment by cwardgar** [comment]
For what it's worth, I've resolved this in #640. But in general, we cannot support thredds builds that aren't built using the `gradlew` wrapper.

**Comment by msdsoftware** [comment]
I was probably doing a `gradle clean build` rather use gradlew because I only intermittently need to compile thredds -- typically when I get a Panoply bug report that tracks back to thredds/NJ and it looks easy enough for me to track down in the source -- and had forgotten about using gradlew to build what I needed.
